app/src/main/res/layout/resume_lesson_activity.xml

```diff
@@ -21,7 +21,7 @@
         android:background="@color/component_color_shared_activity_toolbar_color"
         android:minHeight="?attr/actionBarSize"
         android:theme="@style/Widget.AppCompat.ActionBar"
-        app:navigationContentDescription="@string/hints_andSolution_close_icon_description"
+        app:navigationContentDescription="@string/Navigate up"
```

Comment:
This string format is incorrect, and the app would not have built successfully like this. Generally, please test your changes manually before pushing the code.

Reply:
Done